tests/unit/lms/validation/_api_test.py

```diff
@@ -144,3 +134,18 @@ def all_fields(self):
             "lis_result_sourcedid": "modelstudent-assignment1",
             "score": 0.5,
         }
+
+
+@pytest.fixture
+def json_request(pyramid_request):
```

Comment:
Changing the name of the fixture is a good idea, makes the tests more readable

Reply:
This would be confusing if a test used both `json_request` and `pyramid_request` at once, since `json_request` mutates `pyramid_request`. And is the same object. It could be fixed. But no one does use them both though

Reply:
The fix would be to add a global `make_pyramid_request` fixture that returns a new Pyramid request every time its called. Then change the global `pyramid_request` fixture to depend on `make_pyramid_request`. Then this `json_request` fixture would depend on `make_pyramid_request` as well, not on `pyramid_request`